test/ngAria/ariaSpec.js

```diff
@@ -755,6 +755,38 @@ describe('$aria', function() {
       element.triggerHandler({ type: 'keypress', keyCode: 13 });
       expect(element.text()).toBe('');
     });
+
+    it('should apply role to anchor elements without href ', function() {
+      compileElement('<a ng-click="event = $event">{{event.type}}{{event.keyCode}}</a>');
```

Comment:
Why wouldn't you use a button element for this? I don't understand the use case.